travis.sbt

```diff
@@ -0,0 +1,32 @@
+commands ++= Seq(
```

Comment:
Why a command and no task? Wouldn't a task be easier to use and maintain?

Reply:
At first, I had state changes in both of the commands (adding credentials and changing repo name), therefore I went with a command. Now only the first command does that, therefore the second could be a task. However, having conditional statements would require it to be a DynTask. So I stuck with commands.

Reply:
What's wrong with dyntask?

Reply:
Nothing in particular. It would work for the `publishCi` but not for the `whitesourceCi` which changes the state.

Reply:
But why does it need to update the state? Why not just add the credentials anyway?